docs/docs/launch-manual/sections/core-hacking/sections/building-pulsar.md

```diff
@@ -98,15 +102,13 @@ git submodule init && git submodule update
 Now install and build Pulsar & ppm:
 
 ```sh
-yarn install
-yarn build
-yarn build:apm
+yarn install && yarn build && yarn build:apm
```

Comment:
I'd rather keep this as discrete commands as it will be easier for somebody to tell which point of the process failed.

Reply:
Fixed